packages/@orbit/record-cache/src/live-query/utils.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Record,
+  cloneRecordIdentity,
+  RecordIdentity,
+  RecordOperation
+} from '@orbit/data';
+
+export interface RecordChange extends RecordIdentity {
```

Comment:
I think we'll also need to track changes to `meta` and `links` (which are currently only possible via `addRecord` / `updateRecord` ops).